spec/models/element_to_page_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+module Alchemy
+  describe ElementToPage do
+
+    # ClassMethods
+
+    describe '.table_name' do
+      it "should return table name" do
+        expect(ElementToPage.table_name).to eq('alchemy_elements_alchemy_pages')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.